indexdata.go

```diff
@@ -84,6 +89,34 @@ type indexData struct {
 	repoListEntry RepoListEntry
 }
 
+type symbolData struct {
+	symContent       []byte
+	symIndex         []uint32
+	symKindContent   []byte
+	symKindIndex     []uint32
+	symMetaData      []byte
```

Comment:
so all of these []byte slices are mmap backed right?

Reply:
Yes, they should be. They're referenced using `readSectionBlob`.

```
func (r *indexData) readSectionBlob(sec simpleSection) ([]byte, error) {
	return r.file.Read(sec.off, sec.sz)
}
```

`file` should be `mmapedIndexFile` on linux.